src/Components/Filter/Filter.jsx

```diff
@@ -0,0 +1,65 @@
+import './filter.css'
+const Filter=()=>{
+    return(
+        <div className="flex-column">
+                    <button className="btn-link padding-sm">Clear All Filters</button>  
+                    <hr/>            
+                    <h3 className="padding-sm">Filters</h3>
+                    <h3 className="padding-sm">Price Range</h3>
+                    <input className="padding-sm" type="range" min="1" max="100" value="50"/>
+                    <p className="padding-sm">Price under: 500</p>
+                    <h3 className="padding-sm">Category</h3>
+                    <div className="padding-sm">
+                        <input type="checkbox" id="all categories" name="all categories" checked/>
+                        <label for="all categories">All Categories</label>
+                        <br/>
+                        <input type="checkbox" id="category 1" name="category 1"/>
+                        <label for="category 1">Category 1</label>
+                        <br/>
+                        <input type="checkbox" id="category 2" name="category 2"/>
+                        <label for="category 2">Category 2</label>
+                        <br/>
+                        <input type="checkbox" id="category 3" name="category 3"/>
+                        <label for="category 3">Category 3</label>
```

Comment:
you can make an array of categories and map over it instead  of repeating  code

Reply:
Have made the suggested changes and created an array of products and then mapped over them